extensions/wikipedia/src/wikipedia.ts

```diff
@@ -14,6 +15,20 @@ export interface PageSummary {
   };
 }
 
+interface Preferences {
+  zhVariant: string;
```

Comment:
As this is an optional preference, I believe we should type it as such.

```suggestion
  zhVariant?: string;
```